var/spack/repos/builtin/packages/valgrind/package.py

```diff
@@ -69,11 +73,11 @@ def configure_args(self):
         if not (spec.satisfies('%clang') and sys.platform == 'darwin'):
             # Otherwise with (Apple's) clang there is a linker error:
             # clang: error: unknown argument: '-static-libubsan'
-            options.append('--enable-ubsan')
+            if spec.satisfies('+ubsan'):
+                options.append('--enable-ubsan')
+        if spec.satisfies('+only64bit'):
+            options.append('--enable-only64bit')
```

Comment:
is there also a `--disable` flag for this? If yes, please also explicitly handle the `~only64bit` part

Reply:
I never checked. It was not optional before on macOS.